### PR TITLE
fix: No properties displayed

### LINKED ISSUE #5368: No properties displayed
image
Even after filling out some of the properties, there are none.
- Unfortunately don't have screenshot of that 

nft: https://beta.kodadot.xyz/bsx/gallery/1766361197-1?page=1

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5368 
- [x]  remove carbonless from bsx ref: https://github.com/kodadot/nft-gallery/pull/5311#discussion_r1143212098
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

File: components/gallery/GalleryItemDescription.vue [hunk 1]
```diff
@@ -117,7 +117,14 @@ const properties = computed(() => {
     )
   }
 
-  return nftMetadata.value?.attributes
+  const attributes =
+    nftMetadata.value?.attributes || nftMetadata.value?.meta.attributes || []
+  return attributes.map((attr) => {
+    return {
+      trait_type: attr.trait_type || attr.key,
```
Comment (roiLeo) [inline components/gallery/GalleryItemDescription.vue:124]: `Property 'key' does not exist on type 'Attribute'`

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 32.46 USD/KSM ~ 0.616 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xc9fbf52a23ca0544a982aff4e6dba36e33ee01105d24f4f57b490f6082590502](https://kusama.subscan.io/extrinsic/0xc9fbf52a23ca0544a982aff4e6dba36e33ee01105d24f4f57b490f6082590502)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 5 USD/DOT ~ 4 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x4d05c7d042218502ddf5ddfad720cc568ef8303e5eae7d0a47443919864afef3](https://polkadot.subscan.io/extrinsic/0x4d05c7d042218502ddf5ddfad720cc568ef8303e5eae7d0a47443919864afef3)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**